--- a/configure.ac	2017-06-05 01:36:32.866154468 +0300
+++ b/configure.ac	2017-06-05 01:37:28.150447077 +0300
@@ -2174,11 +2174,11 @@
     with_egl_platforms=$with_platforms
 fi
 
-PKG_CHECK_MODULES([WAYLAND_SCANNER], [wayland-scanner],
-        WAYLAND_SCANNER=`$PKG_CONFIG --variable=wayland_scanner wayland-scanner`,
-        WAYLAND_SCANNER='')
+AC_PATH_PROG([WAYLAND_SCANNER], [wayland-scanner])
 if test "x$WAYLAND_SCANNER" = x; then
-    AC_PATH_PROG([WAYLAND_SCANNER], [wayland-scanner], [:])
+        PKG_CHECK_MODULES([WAYLAND_SCANNER], [wayland-scanner],
+                WAYLAND_SCANNER=`$PKG_CONFIG --variable=wayland_scanner wayland-scanner`,
+                WAYLAND_SCANNER='')
 fi
 
 # Do per-EGL platform setups and checks
