--- a/Source/WebKit2/UIProcess/API/wpe/WPEView.cpp	2017-06-22 11:48:38.967121589 +0300
+++ b/Source/WebKit2/UIProcess/API/wpe/WPEView.cpp	2017-06-22 11:48:50.073936062 +0300
@@ -44,7 +44,7 @@ namespace WKWPE {
 
 View::View(struct wpe_view_backend* backend, const API::PageConfiguration& baseConfiguration)
     : m_pageClient(std::make_unique<PageClientImpl>(*this))
-    , m_size{ 800, 600 }
+    , m_size{ 320, 240 }
     , m_viewStateFlags(WebCore::ActivityState::WindowIsActive | WebCore::ActivityState::IsFocused | WebCore::ActivityState::IsVisible | WebCore::ActivityState::IsInWindow)
     , m_compositingManagerProxy(*this)
 {
